virt/virtual_machines/virt-creating-vms.adoc

```diff
@@ -0,0 +1,57 @@
+:_content-type: ASSEMBLY
+[id="virt-creating-vms"]
+= Creating virtual machines
+include::_attributes/common-attributes.adoc[]
+:context: virt-creating-vms
+:toclevels: 3
+
+toc::[]
+
+You create virtual machines (VMs) from templates, instance types, and `VirtualMachine` manifests.
```

Comment:
"You can create virtual machines (VMs)...." I think that you left off the "can"?